doc/source/serve/model_composition.md

```diff
@@ -7,49 +7,63 @@ This section helps you:
 * compose multiple deployments containing ML logic or business logic into a single application
```

Comment:
We say the word graph in a bunch of places in the docs. Do we want to use the same word here? Like you can build a graph using multiple deployments with ML models/business logic?

Reply:
ehhh I personally want to get away from using the word "graph" because it's confusing (it isn't an explicit concept in the API aside from the DAG API we're trying to deprecate)

Reply:
The word is kind of intuitive though for people to understand. Also used in many other places in the doc. We should either clean up all the other docs or (my preference) refer it as a graph here.

Reply:
application is also used throughout the docs and directly matches the API (i.e., see the key concepts page). agree we should make the verbiage more consistent throughout but we certainly don't need to update all references to the word graph in order to make an incremental improvement here